lib/tokens.js

```diff
@@ -1057,7 +1057,7 @@ async function _getUnfilledBatch({
 }
 
 async function _insertBatch({
-  id, internalId, batchVersion, tokenCount, batchInvalidationCount,
+  id, internalId, batchVersion, tokenCount, batchInvalidationCount = 0,
```

Comment:
Hmm, this doesn't seem like the right fix to me. Wouldn't this create a number of problems or garbage data?

It looks like `batchInvalidationCount` just wasn't passed in here erroneously:

https://github.com/digitalbazaar/bedrock-tokenization/blob/35e7c21dc38f58c2a5a70737e5052573b98951a7/lib/tokens.js#L820-L821

If we add it there (which we should do anyway), does that fix the problem?

Reply:
Yes, this does. Updated.